internal/integration/testdata/mysql/time-type-precision-mysql.txt

```diff
@@ -0,0 +1,45 @@
+# Each test runs on a clean database.
+only mysql
```

Comment:
you should write it as: `only 8 56 57`. Or, let's supported the negate option for `only` (I can add it), and it can be written as:
```
! only maria
```